diff --git a/node_modules/multiaddr/src/convert.js b/node_modules/multiaddr/src/convert.js
index c315201..7ff19ad 100644
--- a/node_modules/multiaddr/src/convert.js
+++ b/node_modules/multiaddr/src/convert.js
@@ -51,6 +51,8 @@ Convert.toString = function convertToString (proto, buf) {
     case 55: // dns6
     case 56: // dnsaddr
     case 400: // unix
+    case 449: // sni
+    case 466: // certhash
     case 777: // memory
       return bytes2str(buf)
 
@@ -84,6 +86,8 @@ Convert.toBytes = function convertToBytes (/** @type {string | number } */ proto
     case 55: // dns6
     case 56: // dnsaddr
     case 400: // unix
+    case 449: // sni
+    case 466: // certhash
     case 777: // memory
       return str2bytes(str)
 
diff --git a/node_modules/multiaddr/src/protocols-table.js b/node_modules/multiaddr/src/protocols-table.js
index 3431af5..e22a952 100644
--- a/node_modules/multiaddr/src/protocols-table.js
+++ b/node_modules/multiaddr/src/protocols-table.js
@@ -45,6 +45,8 @@ Protocols.table = [
   [275, 0, 'p2p-webrtc-star'],
   [276, 0, 'p2p-webrtc-direct'],
   [277, 0, 'p2p-stardust'],
+  [280, 0, 'webrtc-direct'],
+  [281, 0, 'webrtc'],
   [290, 0, 'p2p-circuit'],
   [301, 0, 'udt'],
   [302, 0, 'utp'],
@@ -59,7 +61,12 @@ Protocols.table = [
   [444, 96, 'onion'],
   [445, 296, 'onion3'],
   [446, V, 'garlic64'],
+  [448, 0, 'tls'],
+  [449, V, 'sni'],
   [460, 0, 'quic'],
+  [461, 0, 'quic-v1'],
+  [465, 0, 'webtransport'],
+  [466, V, 'certhash'],
   [477, 0, 'ws'],
   [478, 0, 'wss'],
   [479, 0, 'p2p-websocket-star'],
